libs/platform/icon-tab-bar/components/icon-tab-bar-text-type/icon-tab-bar-text-type.component.html

```diff
@@ -33,6 +33,7 @@
                         [colorAssociations]="colorAssociations()"
                         [attr.tabindex]="idx === 0 ? 0 : -1"
                         [attr.aria-selected]="item.uId === selectedUid"
+                        [attr.aria-level]="3"
```

Comment:
Because the aria-level is configurable for the dynamic page title via the `headingLevel` input, I think this should be `[attr.aria-level]="_headingLevel() + 1"`